README.rst

```diff
@@ -541,6 +554,11 @@ Choose from the list of available fixers:
                         the class name should match
                         the file name. (Risky fixer!)
 
+* **random_api_migration**
```

Comment:
I still think we need an option to enable fixing to that. Especially since the `random_compat` package exists for PHP 5.